test/Database/LSMTree/Model/Normal/Session.hs

```diff
@@ -152,13 +152,16 @@ type TableHandleID = Int
 
 data TableHandle k v blob = TableHandle {
     tableHandleID :: TableHandleID
-  , config        :: SUT.TableConfig
+  , config        :: TableConfig
   }
   deriving Show
 
+data TableConfig = TableConfig
+  deriving Show
```

Comment:
INFO: We don't use the contents of `SUT.Tableconfig` anywhere, so we can define a simple modelled type